src/components/ChefsRestaurantCard/ChefsRestaurantCard.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
```

Comment:
Remove all these redundant imports of `React` as you don't need them anymore.

Reply:
@nitzanpap done